next_rails.gemspec

```diff
@@ -24,10 +24,11 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "colorize", ">= 0.8.1"
+  spec.add_dependency "activesupport", "< 6.0"
```

Comment:
@manuca How come this is necessary now?

Reply:
> @manuca How come this is necessary now?

@etagwerker This is the culprit:
https://github.com/manuca/next_rails/blob/make-actionview-optional/exe/bundle_report#L59

Reply:
@manuca Weird. And where is that code getting called? What methods are we using from active_support?

Reply:
Which to be honest I have no idea why we're depending on it as there are no `acts_like?` calls anywhere as far as I've searched.

For reference this is `acts_like.rb`:
https://github.com/rails/rails/blob/5-2-stable/activesupport/lib/active_support/core_ext/object/acts_like.rb

Reply:
@etagwerker I removed the `activesupport` dependency. Specs pass and `bundle_report` seems to work as expected.

Reply:
@manuca Interesting. I ran into the reference to `acts_like?` in v1.0.3: #21. I guess it is getting used in the exe command.